examples/game_werewolf/README.md

```diff
@@ -62,12 +64,9 @@ With msghub and pipeline, it's very easy to program a discussion among agents.
 
 ### Prompt to Variable
```

Comment:
This section should be updated. For example, `parse_func` and `fault_handler` are gone. `self.parser` is a new parameter. `if x.get("agreement", False)` should be `if x.metadata.get("finish_discussion", False)`.
```
Response in the following format which can be loaded by
json.loads() in python
{{
    "thought": "thought",
    "speak": "thoughts summary to say to others",
    "agreement": "whether the discussion reached an agreement or not (true/false)"
}}
```
 And the above may be changed to some descriptionws about the parser, such as 
```
to_wolves_vote = "Which player do you vote to kill?"

wolves_vote_parser = MarkdownJsonDictParser(
    content_hint={
        "thought": "what you thought",
        "speak": "player_name",
    },
    required_keys=["thought", "speak"],
    keys_to_memory="speak",
    keys_to_content="speak",
)
```

Reply:
README updated according to the comment.